Content.Shared/GameObjects/Components/Body/SharedBodyComponent.cs

```diff
@@ -697,6 +702,19 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 }
             }
         }
+
+        public virtual void Gib(bool recursive = false)
+        {
+            foreach (var (_, part) in Parts)
+            {
+                RemovePart(part);
+
+                part.Owner.RandomOffset(0.5f);
```

Comment:
Wouldn't this always cause a misprediction between the client and server?

Reply:
True, changed now.